pyproject.toml

```diff
@@ -40,3 +40,20 @@ select = [
     # Pyupgrade
     "UP",
 ]
+
+[tool.cibuildwheel]
+build-verbosity = 1
+skip = [
+    "cp38-*",
+    "cp39-*",
```

Comment:
note that it shouldn't be necessary to list versions you *do* (`build`) or *do not* (`skip`) support here; the modern approach is to rely soly on `project.requires-python`, but the `project` table is entirely missing here (the project metadata is instead 100% dynamically generated at build time via `setup.py`). While out of scope here, I could easily contribute a bit of modernization in that area too.